src/utils/index.js

```diff
@@ -0,0 +1,5 @@
+export { filterCategories } from "./filterCategories";
+export { filterBy } from "./filterBy";
+export { priceRange } from "./priceRange";
+export { setRating } from "./setRating";
```

Comment:
suggestion - since the filters are one liner, you can do a filter chaining instead.

Reply:
I thought it will be clear if all filter method logic is kept seperate. will see if I can chain all.